docs/interpreter/jdbc.md

```diff
@@ -853,7 +853,7 @@ Properties
   </tr>
   <tr>
     <td>default.url</td>
-    <td>jdbc:hive2://kyuubi-server:10009</td>
+    <td>jdbc:kyuubi://kyuubi-server:10009</td>
```

Comment:
both `jdbc:hive2//` and `jdbc:kyuubi//` work, but `jdbc:kyuubi//` is recommended